pyhf/readxml.py

```diff
@@ -6,6 +6,15 @@
 import numpy as np
 import tqdm
 
+def extract_error(h):
+    if h.fSumw2:
+        return np.sqrt(h.fSumw2[1:-1]).tolist()
+    else:
+        #if fSumw2 is not filled, it must have
+        #been filled with no weights and .Sumw2() never
+        #got called. therefore error is sqrt(entries)
+        return np.sqrt(h.numpy[0]).tolist()
+
```

Comment:
More pythonic? 

```python
def extract_error(h):
  err = h.fSumw2[1:-1] if h.fSumw2 else h.numpy[0]
  return np.sqrt(err).tolist()
```

Reply:
@lukasheinrich Agree, this would be a good thing to add unless @jpivarski already has this in the works.

I like the slightly more Pythonic version, so I've made a commit.